paradise_papers_django/search/templates/search/display_information/entity.html

```diff
@@ -1,60 +1,168 @@
-<p>Registered in: {{ node_info.jurisdiction_description }}</p>
-<p>jurisdiction: {{ node_info.jurisdiction }}</p>
-<p>Service Provider: <strong>{{ node_info.service_provider }}</strong></p>
-<p>Linked countries: {{ node_info.countries }}</p>
-<hr>
-
-{% if others %}
-    <div>
-        <span>Connected to <strong>{{ others|length }} Others</strong></span>
-    </div>
-{% endif  %}
-
-{% if addressess %}
-    <div>
-        <span>Connected to  <strong>{{ addressess|length }} Address</strong></span>
-    </div>
-{% endif  %}
-
-{% if officers %}
-    <div>
-        <span>Connected to  <strong>{{ officers|length }} officers</strong></span>
-    </div>
-{% endif  %}
-
-{% if intermediaries %}
-    <div>
-        <span>Connected to  <strong> {{ intermediaries|length }} intermediaries</strong></span>
-    </div>
-
-    <h4>its Intermediary:</h4>
-    {% for intermediary in intermediaries %}
-        <h3>{{ intermediary.name }}</h3>
-        <p>Status: {{ intermediary.status }}</p>
-        <p>Countries: {{ intermediary.countries }}</p>
-    {% endfor %}
-    <hr>
-{% endif  %}
-
-{% if officers %}
-    <h4> its Officer:</h4>
-    {% for off in officers %}
-        <ul>
-            <li>{{ off.name }}</li>
+{% load nodes_extra %}
+<div class="node">
+   <header>
+      <h3>Entity</h3>
+      <h1 class="company-title"> {{ node_info.name }} </h1>
+   </header>
+   <div class="pure-g">
+      <div class="left-side">
+         <ul class="relationships">
+            {% if entityC %}
+            <li>
+               <span>  Connected to  <strong> {{ entityC|length }} entity</strong></span>
+            </li>
+            {% endif  %}
+            {% if others %}
+            <li >
+               <span>  Connected to  <strong> {{ others|length }} Others</strong></span>
+            </li>
+            {% endif  %}
+            {% if addressess %}
+            <li>
+               <span>  Connected to  <strong> {{ addressess|length }} Address</strong></span>
+            </li>
+            {% endif  %}
+            {% if officers %}
+            <li>
+               <span> Connected to  <strong> {{ officers|length }} officers</strong></span>
+            </li>
+            {% endif  %}
+            {% if intermediaries %}
             <li>
-                <ul>countries: {{ off.countries }}</ul>
+               <span>  Connected to  <strong> {{ intermediaries|length }} intermediaries</strong></span>
             </li>
+            {% endif  %}
+         </ul>
+         <ul>
             <li>
-                <ul>source: {{ off.sourceID }}</ul>
+               <span>Incorporated: </span> <strong>{{ node_info.incorporation_date }}</strong>
             </li>
-        </ul>
-    {% endfor%}
-{% endif  %}
+            <li>
+               <span>Status: </span> <strong>{{ node_info.status }}</strong>
+            </li>
+            <li>
+               <span>Registered in: </span> <a href="#">{{ node_info.jurisdiction_description }}</a>
```

Comment:
🤔 If it doesn't redirect to other page; can this be a button?
What you thins?

Reply:
It will redirect , we have to implement "search by jurisdiction " first